src/main/java/com/device/shop/entity/Product.java

```diff
@@ -0,0 +1,36 @@
+package com.device.shop.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "products")
+@Data
```

Comment:
avoid using Lombok Data annotation in Entity classes - https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/